devel/gettext/Portfile

```diff
@@ -182,29 +177,19 @@ subport ${name}-tools-libs {
     # https://savannah.gnu.org/bugs/?58669
     configure.args-append   --disable-libasprintf
 
+    # Install from the tools subpackage, so that only its files
+    # are installed
+     destroot.dir           ${worksrcpath}/${name}-tools
+
     post-destroot {
         # Remove the executables and the libraries provided
         # by the runtime. Just leave the libraries and headers
         # for the tools.
-        delete {*}[glob ${destroot}${prefix}/*/libintl.*] \
-            {*}[glob ${destroot}${prefix}/bin/*] \
+        delete {*}[glob ${destroot}${prefix}/bin/*] \
             {*}[glob ${destroot}${prefix}/share/*]
```

Comment:
You have removed the portfile code that deleted libraries but you haven't removed the comment that says libraries get deleted.